lib/recurly/response.php

```diff
@@ -57,7 +57,8 @@ public function assertValidResponse()
       case 0:
         throw new Recurly_ConnectionError('An error occurred while connecting to Recurly.');
       case 400:
-        $message = (is_null($error) ? 'Bad API Request' : $error->description);
+        $details = $this->details ? ' Details: ' .$this->details : '';
```

Comment:
It seems odd to be referencing `$this` rather than `$error`? Though in either case if it runs through `parseErrorXml` I'd expect an error object to come back and that we could use the interpolated value in `$error->message`. How were you testing this out?

Reply:
I think you were right, that was supposed to be `$error` rather than `$this` in that context

Reply:
You're right, there is no unit test for this change. I'll add a test in the `Tests` directory.